### ISSUE TITLE
tile collapsing detection can be improved

### ISSUE THREAD

**Issue by falquaddoomi** [BRCAChallenge/brca-exchange#862]
TL;DR: I forgot to include a small fix in PR #851 to how panels detect their collapsed state. I'll be PRing the fix shortly.

The current master version of `onChangeGroupVisibility()` in `VariantDetail` uses a relatively fragile DOM-traversing method for detecting whether the panel on which the function is invoked is in the process of collapsing, which is then used to set the collapsed state for the group in local storage. (This breaks for the ISPP tile, causing it to always appear collapsed.) I've written and tested out a fix that relies on the previous collapsed state of the group in local storage, which should be much more robust to changes in the panel's DOM representation.

(I thought I'd already included it, actually, but I guess I must have stashed it and forgotten to reapply it before all the big layout fixes in PR #851.)
